packages/cardano-services/test/jest-setup/rebuild-test-db.sh

```diff
@@ -10,7 +10,7 @@ DB_DB_SYNC=$(cat $SECRETS_DIR/postgres_db_db_sync)
 USER=$(cat $SECRETS_DIR/postgres_user)
 PASSWORD=$(cat $SECRETS_DIR/postgres_password)
 
-export DB_SYNC_CONNECTION_STRING="postgresql://${USER}:${PASSWORD}@localhost:5435/${DB_DB_SYNC}"
+export DB_SYNC_CONNECTION_STRING="postgresql://${USER}:${PASSWORD}@localhost:5432/${DB_DB_SYNC}"
```

Comment:
I would avoid to use the standard postgres port; if a developer has another postgres instance it may be source of problems,

Reply:
Done: 87faa2f7068fa9426a49798a4ecd8230a311607b